### PR TITLE
Feature/us 207/account verification

### PR DESCRIPTION
**Pull request by VillalobosGarciaGaby**
## Author
@JeferssonCL 
@denisgandel 
@VillalobosGarciaGaby 
@EmanuelJGalindo 
@zrock-dev 

## What?
- Implemented functionality to the account verification UI so that the user can request to be "Administrator". 
- Performed account authentication, in administrator mode where an email account from hermes.info.app@gmail.com gets to see all users requesting to be administrators.
- I sent an email to hermes.info.app@gmail.com notifying who is the user applying for admin.
- In case he does not want to be an administrator he will be allowed to continue with a general account.
- The verification code is random.

## Why?
- Have a user in administrator mode, so that hermes itself can manage.
- Give the user the option to become an administrator and create possible crashes.

## Code Reviewers
@saletG08 
@SamuelEscalera 
@Karvlox 

## Trainer 
@mmoralesv

### CODE REVIEW

**Review by Karvlox — APPROVED** [review]

**Review by SamuelEscalera — APPROVED** [review]

File: app/src/main/java/com/isc/hermes/controller/PopUp/PopUpContinueLikeGeneralUser.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Build;
+import android.view.View;
+import com.isc.hermes.MainActivity;
+import com.isc.hermes.R;
+import com.isc.hermes.database.AccountInfoManager;
+import com.isc.hermes.model.User.UserRepository;
+
+public class PopUpContinueLikeGeneralUser extends PopUp{
```
Comment (saletG08) [inline app/src/main/java/com/isc/hermes/controller/PopUp/PopUpContinueLikeGeneralUser.java:17]: This class don't have JavaDocs, add JavaDocs.

File: app/src/main/java/com/isc/hermes/controller/PopUp/PopUpWarningIncorrectData.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import android.view.Gravity;
+import android.view.View;
+import android.widget.LinearLayout;
+
+import com.isc.hermes.MainActivity;
+import com.isc.hermes.R;
+import com.isc.hermes.database.AccountInfoManager;
+import com.isc.hermes.model.User.UserRepository;
+
+public class PopUpWarningIncorrectData extends PopUp{
```
Comment (saletG08) [inline app/src/main/java/com/isc/hermes/controller/PopUp/PopUpWarningIncorrectData.java:20]: This class don't have JavaDocs, add JavaDocs.

File: app/src/main/java/com/isc/hermes/controller/PopUp/PopUpContinueLikeGeneralUser.java (hunk 1 above)
Comment (VillalobosGarciaGaby) [reply app/src/main/java/com/isc/hermes/controller/PopUp/PopUpContinueLikeGeneralUser.java:17]: Done

File: app/src/main/java/com/isc/hermes/controller/PopUp/PopUpWarningIncorrectData.java (hunk 2 above)
Comment (VillalobosGarciaGaby) [reply app/src/main/java/com/isc/hermes/controller/PopUp/PopUpWarningIncorrectData.java:20]: done

**Comment by saletG08** [comment]
I don't have any comments.

**Review by saletG08 — APPROVED** [review]

**Pull request merged**